docs/cn/contributor/Building-Alluxio-From-Source.md

```diff
@@ -7,48 +7,72 @@ priority: 0
 ---
 
 * 内容列表
-{:toc}
+  {:toc}
 
-## 编译Alluxio
+该指南介绍如何克隆Alluxio仓库, 编译Alluxio源码, 并且在你的环境中运行测试。
 
-该指南介绍如何从头编译Alluxio。
+## 软件版本
 
-这部分内容的前提条件是你已安装Java JDK 8或以上、Maven 3.3.9或以上。
+- [Java JDK 8或以上](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Maven 3.3.](http://maven.apache.org/download.cgi)
+- [Git](https://git-scm.org/downloads)
 
 ### Checkout源码
 
 从Github上获取主分支并编译：
 
-```console
```

Comment:
Why you change to use shell language other than console?

Reply:
`shell` has been changed to`console`